monochart/templates/hpa.yaml

```diff
@@ -0,0 +1,41 @@
+{{- $root := . -}}
+{{- if .Values.hpa.enabled }}
```

Comment:
```suggestion
{{- if .Values.hpa }}
{{- if .Values.hpa.enabled }}
```

Reply:
I think the first line is needed just in case the `hpa` key doesn't exist at all in the helmfile -- which would be the case for every one of our current projects.